src/vmm/src/cpu_config/aarch64/custom_cpu_template.rs

```diff
@@ -217,7 +217,7 @@ mod tests {
                         {
                             "addr": "j",
                             "bitmap": "0bx00100xxx1xxxx00xxx1xxxxxxxxxxx1"
-                        },
+                        }
```

Comment:
I noticed it when I was adding `test_correct_json` test. I copied json string from `test_malformed_json`, fixed it, but the test was still failing. 
Overall these commas did not affect `test_malformed_json` test, because parsing was failing as expected in any case.

Reply:
ah I see, thanks. Do you think we should test the error type rather than just checking the result is `Err`.

Reply:
If you look at the tests, we do check for errors. Overall I am not a big fan of the tests we have for cpu templates. We should refactor them in some future.